frontend/src/_vars.scss

```diff
@@ -0,0 +1,10 @@
+$breakpoint: 1024px;
+$company_color: #003da7;
+$background_first_color: #fafafa;
+$background_second_color: #003da7;
+$background_thrd_color: #fff;
```

Comment:
Das ist glaube ich nicht IE kompatibel aber stellt bei unserem Projekt wahrscheinlich sowieso kein Problem dar haha

Reply:
Native CSS Variablen gehen in IE tatsächlich nicht. Die SASS Variablen werden beim Build durch die Werte ersetzt, das ist dann auch IE-ready 😅